WcaOnRails/app/views/incidents/_incident_tag.html.erb

```diff
@@ -0,0 +1,14 @@
+<%
+tag ||= ""
+reg = Regulation.find_or_nil(tag)
+label_class = reg ? "primary" : "default"
+title = reg ? "#{link_to(reg["id"], reg["url"], target: "_blank")}" : tag
+content = "<a href='#' onclick='resetSelectedTags(event, \"#{tag}\");'>Search incidents with this tag</a>"
```

Comment:
Not sure if it's only me, but I'm used to avoiding `onclick`, maybe you can achieve the same with some separate piece of JavaScript living in some `.js` file?
E.g.
```javascript
$('[data-incident-tag]').on('click', function(event) {
  /* resetSelectedTags function getting the tag name from the data-incident-tag attribute */
});
```

Reply:
Unfortunately I don't have time to go through the PR right now, but I'll reply on this particular comment since you may have ideas that I could use when working on it later.
@jfly: sure thing, I can rename the function :)

@jonatanklosko: the thing is, this html is used in a popover, and basically the html code looks like this:
```html
<span ... data-content="Face Moves: &lt;hr/&gt;&lt;a href=&#39;#&#39; onclick=&#39;resetSelectedTags(event, &quot;12a1&quot;);&#39;&gt;Search incidents with this tag&lt;/a&gt;" data-trigger="focus" data-html="true">12a1</span>
```

I initially did what you suggested but I think the element is not in the DOM yet (it's only added after clicking to open the popover), so unfortunately it doesn't work :(
I came up with this workaround, maybe you have a better solution?

Reply:
Ah, a bit more tricky indeed, if I'm right the popover is recreated on each click. You could try something like this
```js
$('.incident-tag[data-toggle="popover"]').on('shown.bs.popover', function() {
  $('[data-incident-tag]').on('click', function(event) {
    /* resetSelectedTags function getting the tag name from the data-incident-tag attribute */
    event.preventDefault();
    $('.search input').val("");
    var selectize = $('input#incident-tags')[0].selectize;
    selectize.clear();
    var tag = $(this).data('incidentTag');
    selectize.addItem(tag);
  });
});

```

Reply:
Another solution to this might take advantage of event bubbling by placing a click listener on the body. However, it feels like the best solution is @viroulep's original implementation.